.github/workflows/esp32.yml

```diff
@@ -0,0 +1,56 @@
+name: CI for ESP32
+
+on:
+  workflow_call:
+
+env:
+  CARGO_INCREMENTAL: 0
```

Comment:
This comment suggests that `Swatinem/rust-cache` does this already: https://github.com/dtolnay/rust-toolchain/issues/26

Reply:
yes i saw that, honestly I want to leave it there in case it changes downstream